dbms/programs/client/Client.cpp

```
@@ -571,7 +571,8 @@ class Client : public Poco::Util::Application
                     Poco::File(history_file).createFile();
             }
 
-            loop();
+            signal(SIGINT, signalHandler);
```

Error is not checked.